tcbuilder/backend/bundle.py

```diff
@@ -87,7 +87,7 @@ def get_client(self):
         """Create an instance of the Docker client"""
         return docker.from_env()
 
-    def save_tar(self, output_file):
+    def save_tar(self, output_file, compress=True):
```

Comment:
For my understanding: what's the purpose of not compressing the image?

Reply:
To give more context: @microhobby originally removed the `docker-storage.tar` compression for raw images during the `build` command (if passing a docker-compose file) and I decided to keep it.

The reasoning that I see is this: When using a compose file to bundle container images to a raw OS image with the `build` command, TCB generates a temporary bundle directory with the tarred container images (`docker-storage.tar` file) during the bundling step itself. Immediately after that, the container images are unpacked inside the raw image rootfs during the combine step.

If the tarfile is compressed, the unpack step takes considerably longer compared to an uncompressed file. Given that the main reason to compress the file is to save on disk space, it doesn't seem particularly useful to add compression to a temporary file that would be deleted at the end of the command either way.

For TEZI images the compression is still relevant as the docker-storage tarfile itself will be copied to the output image directory.

That being said, I do think that compression on `docker-storage.tar` for raw images should be done if the bundle directory is not temporary i.e. kept on disk, like when using the commands individually (`bundle` then `combine`) or if passing an already existing bundle directory instead of a docker-compose file during the `build` command.

Reply:
Agree.